docassemble/ALToolbox/al_income.py

```diff
@@ -38,6 +39,20 @@
 ]
 
 
+def _currency_float_to_decimal(value: Union[str, float]) -> Decimal:
+    """Given a float (that was set by a docassemble currency datatype, so truncated at 2 decimal places)
```

Comment:
After a second look, I understand what you're saying here, but initially I thought you were saying, "We're accepting a value that's already been truncated and then we're truncating it." No suggestions at the moment, though. It's a very confusing concept.

Reply:
As confusing as it is, it's kinda true? Docassemble does round (not truncate, I'll change that wording. I was using truncate to mean get rid of additional decimal places, not necessarily to mean `Math.floor` like it sometimes is) the value to the nearest hundredth place, but it's still storing the value in a floating point variable, which isn't exact. So we round it again, and store the rounded value in a variable that can represent it exactly.